src/components/Integrations/Shopify/Shopify.test.js

```diff
@@ -205,4 +215,44 @@ describe('Shopify Component', () => {
       expect(getByText(listExist.value.amountSubscribers.toString()));
     });
   });
+
+  it(`should execute get shopify data every ${
+    FETCH_SHOPIFY_DATA_INTERVAL / 1000
+  } seconds`, async () => {
+    const getShopifyDataMock = jest.fn().mockResolvedValue(oneShopConnected);
+    const getListDataMock = jest.fn().mockResolvedValue({
+      success: false,
+    });
+
+    const shopifyClientDouble = {
+      getShopifyData: getShopifyDataMock,
+    };
+    const dopplerApiClientDouble = {
+      getListData: getListDataMock,
+    };
+    const { container, getByText } = render(
+      <AppServicesProvider
+        forcedServices={{
+          shopifyClient: shopifyClientDouble,
+          dopplerApiClient: dopplerApiClientDouble,
+        }}
+      >
+        <DopplerIntlProvider>
+          <Shopify />
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+
+    const loader = screen.getByTestId('loading-box');
+    await waitForElementToBeRemoved(loader);
+
+    // because it's executed to start
+    expect(getShopifyDataMock).toHaveBeenCalledTimes(1);
+
+    jest.advanceTimersByTime(FETCH_SHOPIFY_DATA_INTERVAL);
+    expect(getShopifyDataMock).toHaveBeenCalledTimes(2);
+
+    jest.advanceTimersByTime(FETCH_SHOPIFY_DATA_INTERVAL);
+    expect(getShopifyDataMock).toHaveBeenCalledTimes(3);
+  });
 });
```

Comment:
This was the missed test @andresmoschini

Reply:
👏👏👏